customers/schema/queries.py

```diff
@@ -146,16 +164,27 @@ class Query:
         unmarked_winter_storage_lease_customer=graphene.Boolean(),
         unmarked_winter_storage_areas=graphene.List(graphene.String),
         sticker_number=graphene.String(),
-        description="The `invoicingTypes` filter takes a list of `InvoicingType` values "
-        "representing the desired invoicing types of the customers. If an empty list is "
-        "passed, no filter will be applied and all the results will be returned."
-        "\n\nThe `customerGroups` filter takes a list of `CustomerGroup` values "
-        "representing the desired groups of the customers. If an empty list is "
-        "passed, no filter will be applied and all the results will be returned."
-        "\n\n**Requires permissions** to access customer profiles."
-        "\n\nErrors:"
-        "\n* A value passed is not a valid invoicing type"
-        "\n* A value passed is not a valid customer group",
+        first_name=graphene.String(),
```

Comment:
Maybe the description on these Helsinki profile fields could indicate that they're the Helsinki profile related fields.